redwood-treehouse-guest/src/commonMain/kotlin/app/cash/redwood/treehouse/StandardAppLifecycle.kt

```diff
@@ -60,9 +61,14 @@ public class StandardAppLifecycle(
   internal val coroutineScope = CoroutineScope(coroutineExceptionHandler)
 
   override fun start(host: Host) {
+    check(!started) { "already started" }
```

Comment:
idea to serve as an input: no-op / return directly if the app already started vs. throw an exception, it makes it potentially less painful when downstream is implemented with flickering state (like session manager 😅)

Reply:
I like to fail fast by default, and relax as necessary later. It reduces the number of different states this thing can be in.

In this case I don’t want to think about what happens if I get two different `Host` instances. That would suck.

Reply:
> I don’t want to think about what happens if I get two different Host instances. That would suck

that's really true!